website/docs/plugin/testing/acceptance-tests/configuration.mdx

```diff
@@ -0,0 +1,277 @@
+                /* ... */
+            },
+        },
+    })
+}
+```
+
+In this instance, the testing configuration is expected to be in the `testdata/directory_containing_config` directory relative to the file containing the `Test_ConfigDirectory_StaticDirectory` test.
+
+### TestNameDirectory
+
+The [TestNameDirectory()](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/config#TestNameDirectory) function will use the name of the executing test to specify a path to a directory containing Terraform configuration. 
+
+For example:
+
+```go
+func Test_ConfigDirectory_TestNameDirectory(t *testing.T) {
+    t.Parallel()
+
+    Test(t, TestCase{
+        Steps: []TestStep{
+            {
+                ConfigDirectory: config.TestNameDirectory(),
+                /* ... */
+            },
+        },
+    })
+}
+```
+
+In this instance, the testing configuration is expected to be in the `testdata/Test_ConfigDirectory_TestNameDirectory` directory relative to the file containing the `Test_ConfigDirectory_TestNameDirectory` test.
+
+### TestStepDirectory
+
+The [TestStepDirectory()](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/config#TestStepDirectory) function will use the name of the executing test and the current test step number to specify a path to a directory containing Terraform configuration.
+
+For example:
+
+```go
+func Test_ConfigDirectory_TestStepDirectory(t *testing.T) {
+    t.Parallel()
+
+    Test(t, TestCase{
+        Steps: []TestStep{
+            {
+                ConfigDirectory: config.TestStepDirectory(),
+                /* ... */
+            },
+        },
+    })
+}
+```
+
+In this instance, because this is the first test step in the test, the testing configuration is expected to be in the `testdata/Test_ConfigDirectory_TestStepDirectory/1` directory relative to the file containing the `Test_ConfigDirectory_TestStepDirectory` test.
+
+## TestStep ConfigFile
+
+The `TestStep.ConfigFile` field accepts a [TestStepConfigFunc](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/config#TestStepConfigFunc) which is a function that accepts a [TestStepConfigRequest](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/config#TestStepConfigRequest) and returns a string containing a path to a file containing Terraform configuration. The path can be a relative or absolute path.
+
+There are helper methods available for generating a [TestStepConfigFunc](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/config#TestStepConfigFunc) including:
+
+* [StaticFile(file string)](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/config#StaticFile)
+* [TestNameFile(file string)](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/config#TestNameFile)
+* [TestStepFile(file string)](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-testing/config#TestStepFile)
+
+~> **Note**: `ExternalProviders` cannot be specified when using `ConfigFile`.
```

Comment:
Same note as above: https://developer.hashicorp.com/terraform/language/providers/requirements#requiring-providers

Reply:
Good point. Have added a sentence to make this clearer and used the link you supplied. Thanks